datadog_checks_dev/README.md

```diff
@@ -217,6 +218,22 @@ Options:
   -h, --help  Show this message and exit.
 ```
 
+#### Create
+
+```console
+$ ddev create -h
+Usage: ddev create [OPTIONS] NAME
+
+  Create a new integration.
```

Comment:
This doesn't actually _create_ a new Integration - it sets up the framework for a new Integration. We should be clear about that here and in the `Options:` below.